docs/hr/content/docs/reusable_snippets/insert_data.ipynb

```diff
@@ -42,7 +42,7 @@
     "from superduperdb import Document\n",
     "\n",
     "def do_insert(data):\n",
-    "    db.execute(table_or_collection.insert([Document({'id': str(idx), 'x': x}) for idx, x in enumerate(data)]))"
+    "    db.execute(table_or_collection.insert([Document({'id': str(idx), 'x': x['x'], 'y': x['y']}) for idx, x in enumerate(data)]))\n"
```

Comment:
Are you sure this will work in case of other use case ?
so this `data` variable here is seeded by
`get_sample_data` snippet

Please check if it is compatible